<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::BeneficiaryManager</class>
 <widget class="QWidget" name="tt3::gui::BeneficiaryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="beneficiariesTreeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="modifyBeneficiaryPushButton">
        <property name="text">
         <string>Modify beneficiary</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/ModifyBeneficiarySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ModifyBeneficiarySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="createBeneficiaryPushButton">
        <property name="text">
         <string>Create beneficiary</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/CreateBeneficiarySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreateBeneficiarySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="destroyBeneficiaryPushButton">
        <property name="text">
         <string>Destroy beneficiary</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/DestroyAccountSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/DestroyAccountSmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>beneficiariesTreeWidget</tabstop>
  <tabstop>createBeneficiaryPushButton</tabstop>
  <tabstop>modifyBeneficiaryPushButton</tabstop>
  <tabstop>destroyBeneficiaryPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>beneficiariesTreeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_beneficiariesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beneficiariesTreeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_beneficiariesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createBeneficiaryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_createBeneficiaryPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyBeneficiaryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_modifyBeneficiaryPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destroyBeneficiaryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_destroyBeneficiaryPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::BeneficiaryManager</receiver>
   <slot>_filterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_beneficiariesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>_beneficiariesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
  <slot>_createBeneficiaryPushButtonClicked()</slot>
  <slot>_modifyBeneficiaryPushButtonClicked()</slot>
  <slot>_destroyBeneficiaryPushButtonClicked()</slot>
  <slot>_filterLineEditTextChanged(QString)</slot>
 </slots>
</ui>
